tracer/test/test-applications/integrations/Samples.Couchbase/Properties/launchSettings.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "profiles": {
+    "Samples.Couchbase": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "COR_ENABLE_PROFILING": "1",
+        "COR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+        "COR_PROFILER_PATH": "$(SolutionDir)tracer\\bin\\tracer-home\\win-$(Platform)\\Datadog.Trace.ClrProfiler.Native.dll",
+
+        "CORECLR_ENABLE_PROFILING": "1",
+        "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+        "CORECLR_PROFILER_PATH": "$(SolutionDir)tracer\\bin\\tracer-home\\win-$(Platform)\\Datadog.Trace.ClrProfiler.Native.dll",
+
+        "DD_DOTNET_TRACER_HOME": "$(SolutionDir)tracer\\bin\\tracer-home",
+        "DD_INTEGRATIONS": "$(SolutionDir)tracer\\bin\\tracer-home\\integrations.json",
+        "DD_VERSION": "1.0.0",
+
+        "DD_TRACE_NETSTANDARD_ENABLED": "true",
+        "DD_TRACE_CALLTARGET_ENABLED" : "true",
+        "DD_CLR_ENABLE_INLINING" : "true"
```

Comment:
These aren't required in `release/2.0`. If we merge this into 1.x, we should remove them subsequently from 2.0, otherwise, if we merge into 2.0, we should remove them now 🙂

Reply:
I assume I'll merge it in 2.0. I don't see the added value on adding this on master. 
So I'll remove those now.

Reply:
I agree, in which case, I suggest changing the base target of the branch, and rebasing onto `release/2.0` 🙂